### DISCUSSION TITLE
Tool Not Working on Twitter's "expandedUrl" with /i/web/status/ tweets

### DISCUSSION THREAD

**Discussion by whileseated** [privatenumber/snap-tweet#35] [Ideas]
I'd like to run this tool to work on the batch of links included in my downloaded twitter archive. The "like.js" file of every tweet I've faved displays links like this: https://twitter.com/i/web/status/1485392929527152641 which differs from the syntax of the @jack example.

The snap-tweet tool doesn't resolve those links, even though they resolve in any browser. Wish it worked! Any idea how to resolve those non-@username links? Thanks - cool tool!

**Comment by privatenumber** [comment]
You can click share to get the normal URL: https://twitter.com/thomaswheatley/status/1485392929527152641

If you want to add support for that URL format, feel free to open a PR.
